rpcs3/rpcs3qt/trophy_manager_dialog.h

```diff
@@ -74,6 +74,7 @@ private Q_SLOTS:
 	bool m_show_silver_trophies = true;
 	bool m_show_gold_trophies = true;
 	bool m_show_platinum_trophies = true;
+	std::string m_TROPHY_DIR;
```

Comment:
why capitals?

Reply:
Probably some old code. Fixed it, see above.